packages/mongo-nav/src/MongoNav.story.tsx

```diff
@@ -24,17 +24,18 @@ storiesOf('MongoNav', module).add('Default', () => (
         activeNav={select(
           'activeNav',
           Object.values(NavElement),
```

Comment:
Hrm....this is now a lie. `NavElement` contains way more than what you can actually set as an activeNav value.

We should probably make a separate object that just contains the values that we'll actually use here, like `ActiveNav` as the object and its values can be taken from `NavElement`.

Reply:
👍 Didn't construct the most beautifully but it does work